bucket/juce.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "5.4.7",
+    "description": "Open-source cross-platform C++ audio application framework",
+    "homepage": "https://juce.com/",
+    "license": {
+        "identifier": "ISC,GPL-3.0-or-later",
+        "url": "https://juce.com/juce-5-license"
+    },    
+    "url": "https://github.com/juce-framework/JUCE/releases/download/5.4.7/juce-5.4.7-windows.zip",
+    "extract_dir": "JUCE",
+    "bin": "Projucer.exe",
```

Comment:
Are there any command line arguments available?

Reply:
Looks like there is -- https://juce.com/discover/stories/projucer-manual#10.4-command-line-tools